src/kkos2-display-bundle/Cron/BrugByenSisCron.php

```diff
@@ -48,16 +46,28 @@ public function getSlideData(SlidesInSlideEvent $event) {
     // Clear errors before run.
     $slide->setOption('cronfetch_error', '');
 
+    try {
+      $this->processSlideEvent($slide);
+    }
+    catch (\Exception $e) {
+      $this->logger->error("Error while processing BrugByenSisCron slide event: " . $e->getMessage());
+      $cronErrors[] = $e->getMessage();
+    }
+  }
+
+  protected function processSlideEvent(SlidesInSlide $slide) {
     $events = [];
     $cronErrors = [];
     $this->eventfeedHelper->setSlide($slide, 'kk-brugbyen');
-    $data = $this->eventfeedHelper->fetchData();
 
-    $data = $this->eventfeedHelper->sliceData($data);
-    foreach($data as $eventData) {
+    $slices = $this->eventfeedHelper->fetchData();
+    $slice = $this->eventfeedHelper->sliceData($slices);
```

Comment:
Related to my comment above I am not really a fan of multilayered exception handling with the handling being about the same.

Can we keep all the exception handling entirely within the `processSlideEvent`function by also wrapping these two lines in a new try/catch?